### PR TITLE
#713 - Add project role filter to project overview list

### PR DESCRIPTION
**Pull request by reckart**
- Added toggle buttons to filter projects by role
- Documented new functionality

### CODE REVIEW

File: inception-ui-core/src/main/java/de/tudarmstadt/ukp/inception/ui/core/dashboard/projectlist/ProjectsOverviewPage.java [hunk 1]
```diff
@@ -98,12 +107,17 @@
     
     private BootstrapFileInputField fileUpload;
     private WebMarkupContainer projectListContainer;
+    private WebMarkupContainer roleFilters;
+    private IModel<Set<PermissionLevel>> activeRoleFilters;
```
Comment (Rentier) [inline inception-ui-core/src/main/java/de/tudarmstadt/ukp/inception/ui/core/dashboard/projectlist/ProjectsOverviewPage.java]: You could use a fancy `EnumSet`, but it is not too important.

**Comment by reckart** [comment]
Make sure you have the very latest WebAnno SNAPSHOTs when building.

**Comment by Rentier** [comment]
I cleaned my installation. Is that not enough?

**Comment by reckart** [comment]
What do you mean by "cleaned"? If you have WebAnno checked out, you need to pull first. Depending on how you build, you may then need to "mvn clean install" WebAnno before INCEpTION can make use of it. If you do not have WebAnno checked out, then building inception with "mvn -U clean install" should help.

**Comment by Rentier** [comment]
It works for me, although I do not like the large gap on the left side between "Create new project" and project List the new selector introduced.

**Comment by reckart** [comment]
Jenkins, can you test this please?

**Pull request merged**